src/brushes/pattern_brush.class.ts

```diff
@@ -73,7 +69,7 @@ export class PatternBrush extends PencilBrush {
       topLeft = path._getLeftTopCoords().scalarAdd(path.strokeWidth / 2);
 
     path.stroke = new Pattern({
-      source: this.source || this.getPatternSrcFunction(),
+      source: this.source || eval(this.getPatternSrcFunction()),
```

Comment:
@asturur what if going on here??
I remember seeing bug reports on this.
I surfaced now that TS checks it.
I suggest refactoring `getPatternSrcFunction` signature to `getPatternSrcFunction(color)` and then we can drop this weird thing replacing the text in the function and evaling it

Reply:
I am going to checkout this change in favor of #8468

Reply:
we need to delete this getPatternSrcFunction. We shouldn't support serialiazing code.
It was an old thing i thought it was gone.